File: LUA/Cap03/c03ex06.lua
-- c03ex06.lua

palavra = "COMPUTADOR"

print("Palavra ........: " .. palavra)
print()
print("Lado esquerdo ..: " .. palavra:sub(1, 3))
print("Centro .........: " .. palavra:sub(4, 7))
print("Lado direito ...: " .. palavra:sub(-3))
print("Reverso ........: " .. string.reverse(palavra))
print("Copias .........: " .. palavra .. palavra .. palavra)
print("Tira pedaco ....: " .. palavra:sub(1, 7))
print("Nova palavra ...: " .. palavra:sub(1, 6) .. "O" .. palavra:sub(8))

io.write("\nPressione <Enter> para encerrar... ")
io.read()

File: PYTHON/Cap03/c03ex06.py
# c03ex06.py

palavra = "COMPUTADOR"

print("Palavra ........:", palavra)
print()

print("Lado esquerdo .:", palavra[:3])
print("Centro ........:", palavra[3:7])
print("Lado direito ..:", palavra[-3:])
print("Reverso .......:", palavra[::-1])
print("Copias ........:", palavra * 3)
print("Tira pedaco ...:", palavra[:7])
print("Nova palavra ..:", palavra[:6] + "O" + palavra[7:])

input("\nPressione <Enter> para encerrar...")
